components/EventList/EventList.js

```diff
@@ -0,0 +1,14 @@
+import EventPreview from "../EventPreview/EventPreview";
+import { EventListWrapper } from "./EventList.styled";
+
+export default function EventList({ events }) {
+  return (
+    <>
+      <EventListWrapper>
+        {events.map((event) => (
+          <EventPreview key={event.id} event={event} />
+        ))}
+      </EventListWrapper>
+    </>
```

Comment:
```suggestion
      <EventListWrapper>
        {events.map((event) => (
          <EventPreview key={event.id} event={event} />
        ))}
      </EventListWrapper>
```

one parent element -> no fragments necessary :)